src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -328,6 +334,7 @@ static boolean isValidStreamingUrl(@Nullable String url)
         /**
          * Used to indicate that the {@link Activity Activity} should display as a custom status
          * in the official client.
+         * <br>This Activity type is <b>read only</b> for bots.
          *
          * @incubating This feature is currently not officially documented and might change
```

Comment:
now that custom status is documented and i'd say "stable", would removing the incubating annotation be okay?

Reply:
Do you have a Link to the documentation?

Reply:
https://discord.com/developers/docs/topics/gateway#activity-object-activity-types

Reply:
It is incubating because bots still can't use it.